helpers.php

```diff
@@ -14,7 +14,7 @@ class KirbyGitHelper
     public function __construct($repoPath = false)
     {
         $this->repoPath = $repoPath ? $repoPath : kirby()->roots()->content();
-        $this->branch = c::get('gcapc-branch', 'master');
+        $this->branch = c::get('gcapc-branch', '');
```

Comment:
I used the empty string as a default because that's what git.php does: https://github.com/Pascalmh/git.php/blob/master/Git.php#L644-L647